compiler/src/compiler_test.go

```diff
@@ -4,8 +4,23 @@ import (
 	"fmt"
 	"os/exec"
 	"testing"
+	"github.com/rs/zerolog"
+	"os"
 )
 
+func initLogger() (zerolog.Logger){
```

Comment:
Shouldn't we send a dummy logger to the compiler when running tests? This makes the tests kind of unreadable.

Reply:
The Veridict isn't been used in main function, so we delete the veridict_test.go